mlflow/resource_model.go

```diff
@@ -0,0 +1,73 @@
+package mlflow
+
+import (
+	"context"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/databrickslabs/terraform-provider-databricks/mlflow/api"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// MLFlowModel defines the parameters that can be set in the resource.
+type Model struct {
+	Name        string    `json:"name" tf:"force_new"`
+	Tags        []api.Tag `json:"tags,omitempty" tf:"force_new"`
+	Description string    `json:"description,omitempty"`
+}
+
+// ResourceDashboard ...
+func ResourceMLFlowModel() *schema.Resource {
+	s := common.StructToSchema(
+		Model{},
```

Comment:
for simplicity and consistently reasons, please move ModelsAPI and all types from `mlflow/api/model.go` to this file.